Additional improvements in support of presentation creation

aeslaughter opened the pull request:
Mainly this patch improved the handling of auto imported wiki content.

moosebuild commented:
Results of testing 325e75cf5272c38629581e2be04a87e23169e0c8 using moose_PR_pre_check recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13383

aeslaughter committed:
Updates to PresentationBuilder in support of workshop (refs #4343)

Conflicts:
	python/PresentationBuilder/slides/RemarkSlide.py
	python/PresentationBuilder/slides/SlideWarehouse.py

moosebuild commented:
Results of testing 244bf87971156b0eeed22db68318972983dc1be5 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13394

moosebuild commented:
Results of testing 244bf87971156b0eeed22db68318972983dc1be5 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13395

moosebuild commented:
Results of testing 244bf87971156b0eeed22db68318972983dc1be5 using moose_PR_app_tests recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13397

moosebuild commented:
Results of testing ba5a5887501f9be6879dca1f27a5222808ef4e37 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13404

moosebuild commented:
Results of testing ba5a5887501f9be6879dca1f27a5222808ef4e37 using moose_PR_app_tests recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13407

moosebuild commented:
Results of testing ba5a5887501f9be6879dca1f27a5222808ef4e37 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13405

aeslaughter committed:
Fixed str/bool logic bug in image settings (refs #4343)

moosebuild commented:
Results of testing cdf5e861d57244e54e4643e9515aa4ef8bfd3231 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13412

moosebuild commented:
Results of testing 244bf87971156b0eeed22db68318972983dc1be5 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13396

moosebuild commented:
Results of testing ba5a5887501f9be6879dca1f27a5222808ef4e37 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13406

moosebuild commented:
Results of testing cdf5e861d57244e54e4643e9515aa4ef8bfd3231 using moose_PR_app_tests recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13415

moosebuild commented:
Results of testing cdf5e861d57244e54e4643e9515aa4ef8bfd3231 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13413

moosebuild commented:
Results of testing cdf5e861d57244e54e4643e9515aa4ef8bfd3231 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.com/view_job/13414
